pandas/tests/io/json/test_pandas.py

```diff
@@ -1389,6 +1389,34 @@ def test_from_json_to_json_table_dtypes(self):
         result = read_json(dfjson, orient="table")
         tm.assert_frame_equal(result, expected)
 
+    def test_to_json_from_json_columns_dtypes(self):
```

Comment:
Can you please link back to the original issues here?

Reply:
Sure, 
44a3935